cms/admin/placeholderadmin.py

```diff
@@ -24,8 +26,6 @@
 from django.views.decorators.clickjacking import xframe_options_sameorigin
 from django.views.decorators.http import require_POST
 
-from six.moves.urllib.parse import parse_qsl, urlparse
-
 from six import get_unbound_function, get_method_function
```

Comment:
From what I can see in the diff six looks to be installed just for these two imports, I assume logic that was for 2 that needs to still work in 3.